app/assets/javascripts/misc/recovery-page-controller.js

```diff
@@ -42,7 +42,7 @@ function handleSubmit(event) {
   const invalidMatches = inputs.reduce(function(accumulator, input, index) {
     const value = input.value;
 
-    if (value === recoveryWords[index].innerHTML.replace(/\s+/, '')) {
+    if (value.toUpperCase() === recoveryWords[index].innerHTML.replace(/\s+/, '').toUpperCase()) {
```

Comment:
is it worth porting any crockford-base32 normalizing code? Ex `.replace('o', '0').replace('1', 'L')` type stuff?